manifests/init.pp

```diff
@@ -100,7 +100,9 @@
   Optional[String] $systemd_beat_log_opts_override                    = undef,
   String $systemd_beat_log_opts_template                              = $filebeat::params::systemd_beat_log_opts_template,
   String $systemd_override_dir                                        = $filebeat::params::systemd_override_dir,
-
+  Boolean $chocolatey_provider                                        = $filebeat::params::chocolatey_provider,
+  Optional[String[1]] $chocolatey_package_name                        = $filebeat::params::chocolatey_package_name,
+  Optional[String[1]] $chocolatey_package_version                     = $filebeat::params::chocolatey_package_version
```

Comment:
Can we reuse the existing `package_ensure` parameter here too?

Reply:
will do, I see currently package name is statically set so I would propose to rename chocolatey_package_name to package_name instead.